JuliaInterface/src/JuliaInterface.c

```diff
@@ -36,6 +35,43 @@ static void handle_jl_exception(void)
         handle_jl_exception();                                               \
     }
 
+jl_module_t * get_module_from_string(char * name)
+{
+    // It suffices to use JULIAINTERFACE_EXCEPTION_HANDLER here, as
+    // jl_eval_string is part of the jlapi, so don't have to be wrapped in
+    // JL_TRY/JL_CATCH.
+    jl_value_t * module_value = jl_eval_string(name);
+    JULIAINTERFACE_EXCEPTION_HANDLER
+    if (!jl_is_module(module_value))
+        ErrorQuit("Not a module", 0, 0);
+    return (jl_module_t *)module_value;
+}
+
+Obj Func_InitializeGAPFFEType()
+{
+    JULIA_GAPFFE_type = (jl_datatype_t *)jl_new_primitivetype(
```

Comment:
```suggestion
    JULIA_GAPFFE_type = jl_new_primitivetype(
```
AFAICT there is no need to cast to `(jl_datatype_t *)` here, the return value of `jl_new_primitivetype ` already is `jl_datatype_t *`